xapi-client.opam

```diff
@@ -4,6 +4,7 @@ authors: [ "xen-api@lists.xen.org" ]
 homepage: "https://github.com/xapi-project/xen-api"
 bug-reports: "https://github.com/xapi-project/xen-api/issues"
 dev-repo: "git+https://github.com/xapi-project/xen-api.git"
+build-env: [[ XAPI_VERSION = "v0.0.0" ]]
```

Comment:
This could be set to opam's `version`, once we drop `master` as the version and start using the actual versions of the packages.

Reply:
yes, we can do this when we pin for 8.2 :+1: